test/test_docker_images.py

```diff
@@ -24,7 +24,7 @@
 def test(tmp_path):
     if utils.platform != "linux":
         pytest.skip("the test is only relevant to the linux build")
-    if platform.machine() not in ["x86_64", "i686"]:
+    if platform.machine() not in ["i686"]:
```

Comment:
You want this test to only run on "i686"?

Reply:
This is just a way to bypass the dockcross test which is irrelevant for now. This will get reverted once the default image are set to `manylinux2014` again (part of the `revert me` commit)